sdks/python/apache_beam/transforms/trigger_test.py

```diff
@@ -850,6 +873,16 @@ def diff(actual, expected):
        | beam.Flatten()
        | beam.ParDo(Check(self.allow_out_of_order)))
 
+    with TestPipeline() as p:
+      # TODO(BEAM-8601): Pass this during pipeline construction.
+      p.options.view_as(StandardOptions).streaming = True
+
+      # We can have at most one test stream per pipeline, so we share it.
+      inputs_and_expected = p | read_test_stream
+      inputs_and_expected | CheckAggregation(beam.GroupByKey())
```

Comment:
Does it worth to also add a line for CombineGlobally too?

Reply:
CombineGlobally is implemented on top of CombinePerKey with a single key, so it wouldn't provide more coverage (and would clutter the implementation to deal with the keyed and no-keyed cases).